tests/test_flake8.py

```diff
@@ -9,7 +9,7 @@
 class Flake8RunnerTest(unittest.TestCase):
     def test_run_flake8(self):
         proj_root = pathlib.Path(__file__).parent.parent
-        paths = [proj_root / dirname for dirname in test_modules]
+        paths = [str(proj_root / dirname) for dirname in test_modules]
```

Comment:
Now that Python 3.5 is EOL, is this still necessary?

Reply:
Could you please clarify, what exactly do you mean? Conversion to a `str`?

Reply:
Yes.